### DISCUSSION TITLE
Icon Request: fa-dishwasher

### DISCUSSION THREAD

**Discussion by fbarletta** [FortAwesome/Font-Awesome#21196] [Icon Requests]
Hello!

I would love to formally request icons for wind (air conditioner) and a dishwasher! We would love to use this for our platform to represent the above mentioned.

Please and Thank you.

Frank @ UPTOP

**Comment by fbarletta** [comment]
Hello -- Shameless self-bump!

**Comment by tagliala** [comment]
air conditioner: #3211 or #1739 
wind: probably #3684 

keeping this open as `fa-dishwasher`

**Comment by boodleboutique** [comment]
I would love a set of washing instruction symbols - temperature, no-iron, no tumble-dry, dishwasher safe, dry clean only etc. e.g. http://www.textileaffairs.com/c-common.htm - maybe not all of them, but at least the most common. Linked to issue #5914

**Comment by just-webdesign** [comment]
+1
I would love a set of washing instruction symbols too

**Comment by CreativeNative** [comment]
+1

**Comment by CreativeNative** [comment]
This ticket is now 7 years old. You have thousands of icons, but the dishwasher is still missing. I would love to buy the pro version, but if this icon is missing, it's useless. We rent vacations homes, so we need this icon. When will it be available?

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
